vms/evm/destination_client.go

```diff
@@ -109,7 +154,7 @@ func NewDestinationClient(
 		"Initialized destination client",
 		zap.String("blockchainID", destinationID.String()),
 		zap.String("evmChainID", evmChainID.String()),
-		zap.Uint64("nonce", nonce),
+		zap.Uint64("nonce", pendingNonce),
```

Comment:
should we report the gap here as well? i.e both the pendingNonce and the current nonce at the time of client initialization?